hnix-store-core/src/System/Nix/Internal/Hash.hs

```diff
@@ -126,7 +126,7 @@ printHashBytes32 :: BS.ByteString -> T.Text
 printHashBytes32 c = T.pack $ concatMap char32 [nChar - 1, nChar - 2 .. 0]
   where
     -- The base32 encoding is 8/5's as long as the base256 digest
-    nChar = fromIntegral $ BS.length c * 8 `div` 5
+    nChar = fromIntegral $ ((BS.length c * 8 - 1) `div` 5) + 1
```

Comment:
See https://github.com/haskell-nix/hnix/commit/5b1117f6b90d547488dea21f7d1a9b85aa36aa02#diff-b5e9db944ee4f0a506a6978743b0752aR152

Reply:
https://github.com/NixOS/nix/blob/6024dc1d97212130c19d3ff5ce6b1d102837eee6/src/libutil/hash.hh#L70

Reply:
First added in d58a11e019813902b6c4547ca61a127938b2cc20 (year 2005).